Gemfile.lock

```diff
@@ -217,7 +217,7 @@ GEM
     msgpack (1.3.1)
     multipart-post (2.1.1)
     nio4r (2.5.8)
-    nokogiri (1.13.9)
+    nokogiri (1.13.10)
```

Comment:
we can leave this here this time, but I think is easier and faster to have a separate PR to fix security vulnerabilities or upgrade gems